code/_onclick/hud/screen_objects.dm

```diff
@@ -89,6 +90,8 @@
 	if(!handcuff_overlay)
 		var/state = (slot_id == slot_r_hand) ? "markus" : "gabrielle"
 		handcuff_overlay = image("icon"='icons/mob/screen_gen.dmi', "icon_state"=state)
+	if(!blocked_overlay)
+		blocked_overlay = image("icon"='icons/mob/screen_gen.dmi', "icon_state"="blocked")

```

Comment:
Already discussed this n coderbus but please make it remove the hand icon completely for absolute immersion instead of blocked overlay